samples/connectedhomeip/lock/src/server/RendezvousServer.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "RendezvousServer.h"
```

Comment:
I guess it's temporary solution and files from src/server will be removed after merging chip-app-server integration PR?

Reply:
Yes, I even mentioned that in the PR description ;)